_layouts/download.html

```diff
@@ -76,4 +76,36 @@ <h3>Download {{ page.name }}</h3>
 
         <script type="text/javascript">detectByUA(document.getElementById("download-combo-state"), "{{ page.base }}"); updateCombo(document.getElementById("download-data"));</script>
     </div>
+    {% if page.dev-files %}
+    <div class="section-header">
+        <div class="section-icon"></div>
+        <h3>Developer Files</h3>
+    </div>
+    <div class="section-item">
+        <div class="content">
+            <p>These files are not required to play.</p>
```

Comment:
Think this can use a bit more text

Reply:
`These files are not required to play, and do not add anything to the game. They can be useful for programmers who want to diagnose crash bugs.`
Only point of confusion I can think of is that "crash" in TT context can also mean two vehicles colliding :D